vlib/v/checker/tests/prefix_expr_decl_assign_err.out

```diff
@@ -0,0 +1,11 @@
+vlib/v/checker/tests/prefix_expr_decl_assign_err.vv:2:5: error: non-name on the left side of `:=`
+    1 | fn main() {
+    2 |     &a := 12
+      |     ^
+    3 |     (*d) := 14
+    4 | }
+vlib/v/checker/tests/prefix_expr_decl_assign_err.vv:1:1: error: non-name `(*d)` on left side of `:=`

```

Comment:
The position in the `*` case is done wrong but it is out of the scope of the PR I think.